Example/ViewController.swift

```diff
@@ -540,6 +540,34 @@ extension ViewController: NavigationViewControllerDelegate {
         dismissActiveNavigationViewController()
         clearNavigationMapView()
     }
+    
+    // MARK: - Offline Navigation Tiles switching
+    
+    func tilesVersioning(wantsSwitchTilesVersion versions: [String], completion: @escaping (String?) -> ()) {
+        let alertController = UIAlertController(title: "Switch to Offline Regions",
+                                                message: "There are next tiles versions available:",
+                                                preferredStyle: .actionSheet)
```

Comment:
For `.actionSheet` you should provide a pivot point so as not to crash on iPads.